### PR TITLE
JHEF405PRO: Make corrections to the target

### PR DESCRIPTION
**Pull request by jhemcu**

### CODE REVIEW

**Comment by nerdCopter** [comment]
@haslinghuis , should OSD positions be removed?

File: configs/default/JHEF-JHEF405PRO.config [hunk 1]
```diff
@@ -98,11 +119,15 @@ set battery_meter = ADC
 set ibata_scale = 170
 set beeper_inversion = ON
 set beeper_od = OFF
+set osd_vbat_pos = 14571
+set osd_ah_sbar_pos = 14542
+set osd_ah_pos = 14414
+set osd_current_pos = 14634
```
Comment (haslinghuis) [inline configs/default/JHEF-JHEF405PRO.config:125]: Please remove as this should be user defined or added using manufacturer preset.

File: configs/default/JHEF-JHEF405PRO.config [hunk 2]
```diff
@@ -78,18 +87,30 @@ dma pin A03 1
...
+feature MOTOR_STOP
+feature LED_STRIP
 feature OSD
 
 # master
 set mag_bustype = I2C
 set mag_i2c_device = 1
 set baro_bustype = I2C
 set baro_i2c_device = 1
+set serialrx_provider = CRSF
```
Comment (haslinghuis) [inline configs/default/JHEF-JHEF405PRO.config:113]: Defined by cloud build option

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

File: configs/default/JHEF-JHEF405PRO.config [hunk 3]
```diff
@@ -100,9 +118,9 @@ set beeper_inversion = ON
...
 set max7456_spi_bus = 3
-set dashboard_i2c_bus = 1
 set flash_spi_bus = 3
 set gyro_1_bustype = SPI
 set gyro_1_spibus = 1
-set gyro_1_sensor_align = CW180
-set gyro_1_align_yaw = 1800
+set gyro_1_sensor_align = CW90
+set gyro_1_align_yaw = 900
+set gyro_2_spibus = 1
```
Comment (haslinghuis) [inline configs/default/JHEF-JHEF405PRO.config:126]: Do we have a dual gyro ?

Comment (jhemcu) [reply configs/default/JHEF-JHEF405PRO.config:126]: Yes. We have a dual gyro. Thanks @haslinghuis I saw your modifications, so I made the changes without rebase. I saw that your modifications were correct and can be merged.

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**